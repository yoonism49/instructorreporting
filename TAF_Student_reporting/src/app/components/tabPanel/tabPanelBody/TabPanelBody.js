// @flow weak

import React      from 'react';
import PropTypes  from 'prop-types';

const TabPanelBody = ({
  children
}) => (
  <div className="panel-body">
    <div className="tab-content">
      {children}
    </div>
  </div>
);

TabPanelBody.propTypes = {
  children: PropTypes.node.isRequired
};

export default TabPanelBody;
