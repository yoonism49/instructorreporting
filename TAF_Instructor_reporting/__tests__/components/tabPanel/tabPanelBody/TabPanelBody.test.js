
'use strict';

import React              from 'react';
import TabPanelBody       from '../../../../src/app/components/tabPanel/tabPanelBody/TabPanelBody';
import renderer           from 'react-test-renderer'; // needed both for snpashot testing but also to prevent errors from enzyme

describe('TabPanelBody component', () => {
  it('renders as expected', () => {
    const component = renderer.create(
      <div>
        <TabPanelBody>
          <p>TabPanelBody children here</p>
        </TabPanelBody>
      </div>
    ).toJSON();
    expect(component).toMatchSnapshot();
  });
});
