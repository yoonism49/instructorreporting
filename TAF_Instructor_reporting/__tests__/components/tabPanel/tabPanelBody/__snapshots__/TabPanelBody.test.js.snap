// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TabPanelBody component renders as expected 1`] = `
<div>
  <div
    className="panel-body"
  >
    <div
      className="tab-content"
    >
      <p>
        TabPanelBody children here
      </p>
    </div>
  </div>
</div>
`;
